edge-modules/ModuleRestarter/ModuleRestarter.csproj

```diff
@@ -34,7 +34,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Devices" Version="1.18.1" />
-    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.20.3-asc-edge" />
+    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.21.3-asc-edge" />

```

Comment:
This should be just 1.21.3. Does this even build?

Reply:
Yes, you are correct. Noticed this error but messed up my commit --amend.